client/components/Profiles/RenderImg.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { ImageStyle, StyleProp, View, Image} from 'react-native';
+import { profilePictures } from '@/constants/profilePictures';
+
+interface RenderImageProps {
+  imageUrl: string | null;
+  style?: StyleProp<ImageStyle>;
+}
+
+const RenderImage = ({ imageUrl, style }: RenderImageProps) => {
+
+    if (!imageUrl) return <View/>
```

Comment:
I think this case will break the app

Reply:
it doesn't on mine. it's simply for if it doesn't exist  for typescript (which it definitely always will as it defaults to avatar 1 if nothing is selected)

Reply:
did you test  `RenderImage` without a defined imageUrl? Because you are returning a closing View tag. So on the component you will have:
```
<View/>
<View/>
```
which even if it doesnt break the app doesnt makse sense to me